lighthouse-core/test/fraggle-rock/config/filters-test.js

```diff
@@ -502,7 +502,7 @@ describe('Fraggle Rock Config Filtering', () => {
       });
     });
 
-    it('should filter out audits and artifacts not in the categories by default', () => {
+    it('should keep audits not in any category by default', () => {
```

Comment:
It seems like this should be unrelated to not connecting when the browser wasn't launched. Why is this needed? It seems like a big config filtering change.

Reply:
This issue slipped by because I didn't duplicate the tests in `index-test.js` for the new fraggle rock runner. One of those tests runs lighthouse in audit mode with a config that has no categories, and asserts that the audits are still run.

It isn't related to the browser connection bit, but it does represent where legacy mode and FR are not doing the same thing. The test was related to audit mode, so I put it in this PR. Looking back I don't think the issue is specific to audit mode so yeah this probably makes sense in a new PR.

Reply:
codecov is alarmed by the test coverage. Does #13867 make up for it?

Reply:
> codecov is alarmed by the test coverage. Does https://github.com/GoogleChrome/lighthouse/pull/13867 make up for it?

It should, but I don't trust codecov. It seems to flag lines of code that I guarantee are tested at some point (e.g. the line at the start of `navigationGather`)